src/test/java/org/folio/oaipmh/service/impl/SetServiceImplTest.java

```diff
@@ -54,9 +57,12 @@
   private static final String NONEXISTENT_SET_ID = "a3bd69dd-d50b-4aa6-accb-c1f9abaada55";
   private static final String TEST_USER_ID = "30fde4be-2d1a-4546-8d6c-b468caca2720";
 
-  private static final String EXPECTED_NOT_FOUND_MSG = String.format("Set with id '%s' was not found", NONEXISTENT_SET_ID);
-  private static final String EXPECTED_ITEM_WITH_ID_ALREADY_EXISTS_MSG = String.format("Set with id '%s' already exists",
+  private static final String EXPECTED_NOT_FOUND_MSG = format("Set with id '%s' was not found", NONEXISTENT_SET_ID);
+  private static final String EXPECTED_ITEM_WITH_ID_ALREADY_EXISTS_MSG = format("Set with id '%s' already exists",
       EXISTENT_SET_ID);
+  private static final String DUPLICATE_KEY_VALUE_VIOLATES_UNIQUE_CONSTRAINT_ERROR_MSG = "duplicate key value violates unique constraint \"%s\"";
```

Comment:
It would be nice if this was defined in one place and reused instead of repeating the definition

Reply:
@craigmcnally, in fartther PR requests i've created a separate class for mock data especially for Set functionality testing and after all request will be approved then i will merge all of them i correct order, so such simple issues will be disappear caused the above mentioned reason.